wormchain/Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM docker.io/golang:1.19.9@sha256:86901bd215ca2b33bdef68f70b999140252169bb55883bb29347334fa00b5b62
+FROM golang:1.22.5
 
 #used for a readiness probe
-RUN apt-get update
-RUN apt install -y netcat
-RUN apt install -y jq
+RUN apt update && \
+    apt-get install -y \
+        build-essential \
+        ca-certificates \
+        curl
```

Comment:
The comment on the original file and the downloads of netcat and jq seem to indicate some sort of "readiness probe" requirement for netcat, did you replace these with curl elsewhere?

Reply:
Good catch, I'll add those back. I must've been under the impression build-essential included those packages as well.